doc/users/plotting/colormaps/grayscale.py

```diff
@@ -23,15 +24,17 @@
 mpl.rcParams['mathtext.sf'] = 'sans'
 mpl.rcParams['mathtext.fallback_to_cm'] = 'True'
 
-# Have colormaps separated into categories: http://matplotlib.org/examples/color/colormaps_reference.html
+# Have colormaps separated into categories:
+# http://matplotlib.org/examples/color/colormaps_reference.html
 
 cmaps = [('Sequential',     ['Blues', 'BuGn', 'BuPu',
                              'GnBu', 'Greens', 'Greys', 'Oranges', 'OrRd',
                              'PuBu', 'PuBuGn', 'PuRd', 'Purples', 'RdPu',
                              'Reds', 'YlGn', 'YlGnBu', 'YlOrBr', 'YlOrRd']),
          ('Sequential (2)', ['afmhot', 'autumn', 'bone', 'cool', 'copper',
-                             'gist_heat', 'gray', 'hot', 'pink',
-                             'spring', 'summer', 'winter']),
+                             'gist_heat', 'gray', 'hot', 'inferno', 'magma',
+                             'pink', 'plasma', 'spring', 'summer', 'viridis',
```

Comment:
By ``Suggested Sequential`` you mean we suggest them because they have perceptual uniformity.

In which case why not call them  ``Perceptually Uniform Sequential``.  That way we make it clear why we suggest these ones, i.e. why they should use these ones over the other sequential colour maps...